app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -10,3 +10,4 @@
 <%= presenter.attribute_to_html(:related_url, render_as: :external_link) %>
 <%= presenter.attribute_to_html(:resource_type, render_as: :faceted) %>
 <%= presenter.attribute_to_html(:source) %>
+<%= presenter.attribute_to_html(:rights_statement) %>
```

Comment:
How is this rendering by default? I wonder if we need to handle rights_statements similar to how we handle licenses, using a new `:rights_statement` renderer? Here's what we use for licenses: https://github.com/projecthydra-labs/hyrax/blob/master/app/renderers/hyrax/renderers/license_attribute_renderer.rb